### ISSUE TITLE
Merge Sort

### ISSUE THREAD

**Issue by jainaman224** [jainaman224/Algo_Ds_Notes#2136]
- [x] C
- [x] C++
- [x] C#
- [x] CoffeeScript
- [x] Dart
- [x] Go
- [x] Java
- [x] JS
- [ ] Kotlin - @shubhammantri1
- [x] Php
- [x] Python
- [x] Readme
- [x] Ruby
- [x] Rust - @m-code12
- [x] TypeScript - @m-code12

**Comment by chandrikadeb7** [comment]
I want to work on PHP under GSSoC'20.

**Comment by jainaman224** [comment]
@chandrikadeb7 For now, you have 2 open issues.

**Comment by chandrikadeb7** [comment]
> @chandrikadeb7 For now, you have 2 open issues.

@jainaman224 I have 1 issue open. The other issue got merged.

**Comment by BhardwajShweta25** [comment]
I would like to work on this issue in C++

**Comment by shubhammantri1** [comment]
I Want to work in Kotlin. @jainaman224

**Comment by p-bucky** [comment]
I want to work on this in Java please assign this to me

**Comment by jainaman224** [comment]
@p-bucky @BhardwajShweta25 The languages that are checked are already implemented.

**Comment by shubhammantri1** [comment]
@SakshiShreya Please review my PR.

**Comment by chandrikadeb7** [comment]
@somya-kapoor @SakshiShreya @raghav-dalmia Please review PR #2542

**Comment by jainaman224** [comment]
@shubhammantri1 Are you working on it?

**Comment by m-code12** [comment]
I can implement this in rust .Please assign it to me @jainaman224 , @asha15 .

**Comment by m-code12** [comment]
@jainaman224 , @somya-kapoor , @singh-shreya6 , @asha15  I have created a pull request . Please review my PR #2760 .

**Comment by m-code12** [comment]
I can implement this in typescript .Please assign it to me @jainaman224 @singh-shreya6 @somya-kapoor @asha15 @raghav-dalmia

**Comment by vanya-24** [comment]
I can implement in python. Please assign it to me.

**Comment by thakran14** [comment]
please assign this to me in php

**Comment by thakran14** [comment]
@jainaman224 a php file is already present. Am I supposed to create and add another?

**Comment by jainaman224** [comment]
No, Thanks for informing. It would be great if you can work on different issue.
